tests/llmcompressor/transformers/sparsification/test_compress_tensor_utils.py

```diff
@@ -356,3 +356,182 @@ def test_model_shared_tensors_gpu(
     test_model_shared_tensors(
         offload, torch_dtype, tie_word_embeddings, device_map, tmp_path
     )
+
+
+@pytest.mark.parametrize(
```

Comment:
Can we add tests to make sure we're landing on the correct compressors i.e a `test_accurate_compressor` chosen test or something?

For Fp8 and Int8?

W8A16 - FP8 - naive
W8A16 - FP8 + Sparse - naive + dense
W8A8 - FP8 - float
W8A8 + Sparse  - float + sparse24

W8A16 Int8 + Sparse - marlin24
W8A8 Int8 + Sparse - int + sparse24

Reply:
Very cumbersome test, but added!